opencensus/common/schedule/__init__.py

```diff
@@ -82,7 +83,7 @@ class QueueExitEvent(QueueEvent):
 class Queue(object):
     def __init__(self, capacity):
         self.EXIT_EVENT = QueueExitEvent('EXIT')
-        self._queue = queue.Queue(maxsize=capacity)
+        self._queue = multiprocessing.Queue(maxsize=capacity)
```

Comment:
`queue.Queue` was created to work in concurrent environments spawned with the `threading` module, which is the behavior of the Azure exporters. With that being said, we probably shouldn't replace the default queue that is being used, since the original use case is supposed to be for the concurrent environments. I suggest adding the ability for the user to configure which type of queue they want by adding an option [here](https://github.com/census-instrumentation/opencensus-python/blob/master/contrib/opencensus-ext-azure/opencensus/ext/azure/common/__init__.py#L119). Then you can create a different type queue accordingly.

Reply:
It does work with threading, however in an environment where multiple processes are being used (see linked bug reports), the queue is not being shared between the different processes. Therefore a multiprocessing.Queue is required to get this to work.